src/main/java/com/cgm/infolab/db/model/UserEntity.java

```diff
@@ -12,41 +12,43 @@ public class UserEntity {
     private String description;
     private UserStatusEnum status;
     private ThemeEnum theme;
+    private long avatarId;
```

Comment:
Userei `Optional<Long>` per `avatarId`. In questo modo è esplicito che potrebbe non esserci e non devi fare i confronti con ID.None